client/src/auth/saga.js

```diff
@@ -38,10 +37,26 @@ export function* handleRegister({ payload: { history } }) {
   }
 }
 
+export function* handleRefreshActiveGames({ payload: { id } }) {
```

Comment:
Shouldn't logic for getting games be in a saga near the `Home` container and not here?